### ISSUE TITLE
Semicolon rule problem

### ISSUE THREAD

**Issue by Kaba2** [palantir/tslint#3645]
### Bug Report

- tslint: 5.8.0
- tsc: 2.6.2
- editor: Visual Studio Code

#### TypeScript code being linted

```ts
f(g((x) => x), (x) => x));
```

with `tslint.json` configuration:

```json
{
    "extends": "tslint:recommended",
    "rules": {
        "comment-format": [true, "check-space"],
        "indent": [true, "tabs", 4],
        "max-classes-per-file": [false, 1]
    }
}
```

#### Actual behavior

`f(g((x) => x), (x) => x); )`

#### Expected behavior

`f(g((x) => x), (x) => x));`

**Comment by aervin** [comment]
I don't think your "expected behavior" example is valid. Semicolon is adding a semi where the call ends. The last closing-paren is not necessary. Perhaps the "code being linted" should look like this
```ts
f(g((x) => x, (x) => x));
```
or
```ts
f(g((x) => x), (x) => x);
```

**Comment by Kaba2** [comment]
You are right; it was just an extraneous parenthesis, which I could not see for some reason.
